# DEBUG 配置可能导致会同时启动2个进程, 这将会导致部分异常, 请注意.
# 该问题来源来 Flask
DEBUG: true
TESTING: true
SQLALCHEMY_ECHO: true
OPEN_SIGN_AUTH: false

SQLALCHEMY_DATABASE_URI: 'mysql://root:123456@127.0.0.1:3306/wallet'
SQLALCHEMY_COMMIT_ON_TEARDOWN: true
SQLALCHEMY_TRACK_MODIFICATIONS: true

#JOBS:
#  - id: scanChain
#    func: 'tasks.scan_chain:run_sync'
#    trigger: cron
#    second: '*/10'
#    minute: '*'
#
#  - id: depositChain
#    func: 'tasks.scan_chain:notify_project'
#    trigger: cron
#    second: '0'
#    minute: '*/1'
#
#  - id: collection
#    func: 'tasks.scan_chain:collection_eth'
#    trigger: cron
#    second: '0'
#    minute: '*/1'
#    hour: '*/1'
##    hour: '*/1'
#
#  - id: render
#    func: 'tasks.scan_chain:render_eth'
#    trigger: cron
#    second: '0'
#    minute: '*/1'
#    hour: '*/1'
#    hour: '*/1'

COLLECTION_ADDRESS: "0xeCB50aEEFE28a662B4aEE0267F66273d5A582928"
COLLECTION_MIN_ETH: 0.004
RENDER_ADDRESS: "0x69ebf7a64a8ce92d07afe7ba810933dce18e75dd"
RENDER_AMOUNT: 0.01
GAS: "0x249f0"
